views/admin/connect.php

```diff
@@ -370,34 +370,15 @@
                                 </tr>
                                 <tr valign="top" class="new-list-creation">
                                     <th scope="row">
-                                        List Type
+
                                     </th>
                                     <td>
-                                        <select id="listType" name="type" class="dropdown-select">

```

Comment:
This is removing the ability to create a list in which we are able to choose the list type. This functionality is now removed.